scipy-1.0/references.bib

```diff
@@ -1275,3 +1275,140 @@ @Book{schroeder:2006:VTK
 edition = 4,
 isbn = {978-1-930934-19-1},
 }
+
+@online{Hugunin-first,
```

Comment:
Thank you for doing this. I won't check all of them at this time; I'd prefer to check through all the references at once as we near the end.

Reply:
ok, rebase / force-push after resolving the merge conflicts in references